cluster/src/main/java/com/hartwig/pipeline/tertiary/cider/CiderOutput.java

```diff
@@ -0,0 +1,22 @@
+package com.hartwig.pipeline.tertiary.cider;
+
+import java.util.Optional;
+
+import com.hartwig.pipeline.StageOutput;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface CiderOutput extends StageOutput {
```

Comment:
Will Cider or Teal output ever be used by a downstream stage? If not we can remove a lot complexity by making these output classes only have the name and no output location

Reply:
Orange report might need them later. Would this mean I should keep them?

Reply:
Yup, sounds good.